mig-agent/persist.go

```diff
@@ -64,6 +67,19 @@ func (p *persistModuleRegister) remove(modname string) {
 
 var persistModRegister persistModuleRegister
 
+// Load the configuration file for a persistent module if it exists, and return it
+// as a JSON byte slice so we can send it from the agent to the module after the
+// module is started. If the configuration file cannot be loaded, just return the
+// config struct for the module uninitialized.
+func getPersistConfig(modname string) (ret interface{}) {
+	cfg := modules.Available[modname].NewRun().(modules.PersistRunner).PersistModConfig()
+	confpath := path.Join(PERSISTCONFIGDIR, modname+".cfg")
+	// An error here isn't fatal, we just continue with cfg as is
+	gcfg.ReadFileInto(cfg, confpath)
+	ret = cfg
+	return
+}
```

Comment:
Two thoughts?
1. Should this code belong to the `modules` package instead, just in case other components need to use it?
2. Should it enforce the `gcfg` format, or leave it up to the module to unmarshal the config into a format of its choice? I'm concerned `gcfg` might limit us a bit too much, but maybe not.